src/components/CollectionList/CollectionList.stories.js

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+
+import CollectionList from "./CollectionList";
+
+import imageFile1 from "../../assets/images/Fall-2020-1.png";
+import imageFile2 from "../../assets/images/Fall-2020-2.png";
+import imageFile3 from "../../assets/images/Spring-2020-1.jpeg";
+import imageFile4 from "../../assets/images/Spring-2020-2.jpeg";
+import imageFile5 from "../../assets/images/Trending-Now-1.jpeg";
+import imageFile6 from "../../assets/images/Trending-Now-2.jpeg";
+import imageFile7 from "../../assets/images/Classics-1.jpeg";
+import imageFile8 from "../../assets/images/Classics-2.jpeg";
+
+const slideData = [
+  {
+    image: imageFile1,
+    label: "LUCA",
```

Comment:
The dress names are not uppercase in the Desktop version. Can you update?

Reply:
Good to go